### PR TITLE
WIP: make it possible to extend process from another one

### PR DESCRIPTION
**Pull request by gregorjerse**
in the same file

### CODE REVIEW

File: resolwe/process/parser.py [hunk 1]
```diff
@@ -111,6 +111,11 @@ def __init__(self, source):
         self.source = source
         self.processes = []
         self.base_classes = set()
+        # The defined classes list is a list of all processes defined in the
+        # given file we have seen so far.
+        # This allows us to define processes that inherit from other processes
+        # defined it the same file.
```
Comment (jkokosar) [inline resolwe/process/parser.py:117]: defined _in_

File: resolwe/process/parser.py [hunk 2]
```diff
@@ -251,6 +265,9 @@ def base_classes(self):
 
         :return: A list of the base classes for the processes.
         """
+
+        print(self._source)
```
Comment (jkokosar) [inline resolwe/process/parser.py:269]: Remove the print

**Review by jkokosar — COMMENTED** [review]
Add CHANGELOG entry

**Comment by gregorjerse** [comment]
A different solution is needed.

**Pull request closed without merging**